frontend/src/pages/ConnectedPages/Playlists/components/PlaylistCard/index.tsx

```diff
@@ -31,7 +30,6 @@ const PlaylistCard: React.FC<IProps> = ({ playlist }: IProps) => {
   };
 
   if (!!newRoom) {
-    ConnectionManager.createInstance(newRoom.id.toString());
     return <Redirect to={`/waitingRoom?roomId=${newRoom.id}`} />;
```

Comment:
If roomId is in cookies, why keeping the param in the url ?
We could set the cookie here and redirect to `/waitingRoom`, without param ?